### PR TITLE
fix bugs about glimpse picking 

### PR DESCRIPTION
**Pull request by mixmix**
what it says on the tin. Happily it was a small typo

### CODE REVIEW

**Comment by mixmix** [comment]
Wip

**Comment by HerveH44** [comment]
Thanks for working on Glimpse draft ! I am pretty busy the next 2 weeks, I'll try to follow what you are doing and try to find the last bugs. Let me know what it looks like and I'll take a look.

**Comment by mixmix** [comment]
I'll work on getting the timeout picks but fixed and then let you know.

After that I'll summarize all the bugs mentioned in the glimpse-draft branch to see where we are. I'll probably work on the "confirm" button. I think I can see how to do that from the code you've written.

**Comment by mixmix** [comment]
@HerveH44 this is ready for review and merge

File: backend/game.js [hunk 1]
```diff
@@ -550,7 +550,7 @@ module.exports = class Game extends Room {
 
       if (this.shouldAddBots()) {
         while (this.players.length < this.seats) {
-          this.players.push(new Bot(this.picksPerPack, this.burnsPerPack, this.id));
+          this.players.push(new Bot(this.picksPerPack, this.cube.burnsPerPack, this.id));
```
Comment (mixmix) [inline backend/game.js:553]: This was a typo causing one bug

File: backend/player/human.js [hunk 2]
```diff
@@ -145,29 +141,35 @@ module.exports = class Human extends Player {
...
       picks: [],
       burns: []
-    }
+    };
 
     this.updateDraftStats(this.draftLog.pack, this.pool);
 
     this.emit("pass", pack);
   }
   handleTimeout() {
```
Comment (mixmix) [inline backend/player/human.js:150]: This was they other area that needed tweaking to get auto - top up of missing picks working

**Comment by ZeldaZach** [comment]
Seems there's a conflict on this one @HerveH44 @mixmix

**Comment by mixmix** [comment]
weird. Merged `glimpse-draft` in (with no conflicts)

**Comment by mixmix** [comment]
Cc @ZeldaZach

**Pull request merged**